tests/Doctrine/Tests/Common/Reflection/ExampleAnnotationClass.php

```diff
@@ -9,6 +9,7 @@
  *   key = "value"
  * )
  */
+const foo = \stdClass::class;
```

Comment:
Not worth the effort, I will rather just remove this line. It was a bonus thing only.